sql/driver/driver.go

```diff
@@ -24,8 +24,14 @@ import (
 
 	"github.com/cockroachdb/cockroach/base"
 	"github.com/cockroachdb/cockroach/util"
+	"github.com/cockroachdb/cockroach/util/stop"
 )
 
+// Stopper exposed to allow tests to properly drain client connections.
+//
+// TODO(pmattis): Is there a better way to expose this?
+var Stopper *stop.Stopper
```

Comment:
It is used by tests to cleanly shut down client goroutines. I'm not really fond of the way this is working but we have restrictions from the `database/sql` interface.